pkgs/development/zig-packages/build-zig-package/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, zig }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
+  inherit strictDeps enableParallelBuilding;
+  nativeBuildInputs = [ zig ] ++ nativeBuildInputs;
+
+  postConfigure = ''
+    export HOME=$TMPDIR
```

Comment:
```suggestion
    export HOME=$(mktemp -d)
```

Reply:
This is what I originally had, but someone told me to change it. 
I will undo this change.